clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -414,8 +495,69 @@ public void testOffsetFetchRequestErroredRequests(final Errors error, final bool
             testNonRetriable(futures);
             assertEmptyPendingRequests(commitRequestManger);
         }
+    }
 
-        assertCoordinatorDisconnect(error);
+    @Test
```

Comment:
nit: Should we use parameterized tests instead of specifying all cases like this?

Reply:
Sure, done for this and other similar cases that could be parametrized (That was the approach, not even sure why I did not use them in these)